src/app/components/client/DataBrokerProfiles.tsx

```diff
@@ -10,12 +10,20 @@ import { useL10n } from "../../hooks/l10n";
 import IconChevronDown from "./assets/icon-chevron-down.svg";
 import { useState } from "react";
 import { OnerepScanResultRow } from "knex/types/tables";
-import { OpenInNew } from "../server/Icons";
+import { DataBrokerNames } from "../../functions/universal/dataBrokerNames";
 
 export type Props = {
   data: OnerepScanResultRow[];
 };
 
+function getDataBrokerName(dataBrokerName: string) {
+  const result = DataBrokerNames.find(
+    ({ data_broker }) => data_broker === dataBrokerName,
+  );
+
+  return result?.data_broker_pretty || dataBrokerName;
```

Comment:
The logic here is that if a data broker is not on our list, we serve the provided name from the OneRep API.